.gitignore

```diff
@@ -2,3 +2,4 @@
 /Cargo.lock
 tarpaulin-report.html
 **/test_snapshots/
+.soroban/
```

Comment:
Why adding this?

Reply:
It is because the commands `soroban config network...` and `soroban keys generate..` in `release.sh` create files in a folder called `.soroban`